app/main.py

```diff
@@ -1,8 +1,50 @@
+import decimal
```

Comment:
The `decimal` module is imported but not used correctly in the type annotations. You should use `float` for the `average_rating` parameter in the `CarWashStation` class, as specified in the task description.